ChocolateyPackage/textanalysistoolnet.portable.nuspec

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What command generates the .nupkg file? Should that live here, too?

Reply:
Good point, added a readme.md file describing this.